lib/arena.rb

```diff
@@ -1,2 +1,49 @@
+
+
 class Arena
+  attr_accessor :name, :gladiators
```

Comment:
could instead to `attr_reader` to prevent adding gladiators without calling the `add_gladiator` method